docs/resources/dws_cluster_v1.md

```diff
@@ -50,61 +50,71 @@ resource "flexibleengine_dws_cluster_v1" "cluster" {
+    characters, which consist of letters, digits, hyphens (-), or underscores(_) only and must start with a letter.
+    Changing this will create a new resource.
 
-* `number_of_node` - (Required) Number of nodes in a cluster. The value ranges
-    from 3 to 32.
+* `node_type` - (Required, String, ForceNew) Node type.Changing this will create a new resource.
 
-* `user_name` - (Required) Administrator username for logging in to a data
+* `number_of_node` - (Required, Int, ForceNew) Number of nodes in a cluster. The value ranges from 3 to 32.
+  Changing this will create a new resource.
+
+* `user_name` - (Required, String, ForceNew) Administrator username for logging in to a data
     warehouse cluster The administrator username must:
     - Consist of lowercase letters, digits, or underscores.
     - Start with a lowercase letter or an underscore.
     - Contain 1 to 63 characters.
     - Cannot be a keyword of the DWS database.
+    Changing this will create a new resource.
 
-* `user_pwd` - (Required) Administrator password for logging in to a data
+* `user_pwd` - (Required, String, ForceNew) Administrator password for logging in to a data
     warehouse cluster. A password must conform to the following rules:
     - Contains 8 to 32 characters.
     - Cannot be the same as the username or the username written in reverse order.
     - Contains three types of lowercase letters, uppercase letters, digits and
       special characters ~!@#%^&*()-_=+|[{}];:,<.>/?
+    Changing this will create a new resource.
 
-* `vpc_id` - (Required) VPC ID, which is used for configuring cluster network.
+* `vpc_id` - (Required, String, ForceNew) VPC ID, which is used for configuring cluster network.
+  Changing this will create a new resource.
 
-* `subnet_id` - (Required) The ID of the VPC Subnet, which is used for configuring cluster network.
+* `subnet_id` - (Required, String, ForceNew) The ID of the VPC Subnet, which is used for configuring cluster network.
+  Changing this will create a new resource.
 
-* `security_group_id` - (Required) ID of a security group. The ID is used for
-    configuring cluster network.
+* `security_group_id` - (Required, String, ForceNew) ID of a security group. The ID is used for
+    configuring cluster network. Changing this will create a new resource.
 
-* `port` - (Optional) Service port of a cluster (8000 to 10000). The default value is 8000.
+* `port` - (Optional, Int, ForceNew) Service port of a cluster (8000 to 10000). The default value is 8000.
+  Changing this will create a new resource.
 
-* `availability_zone` - (Optional) AZ in a cluster.
+* `availability_zone` - (Optional, String, ForceNew) AZ in a cluster.Changing this will create a new resource.
 
-* `public_ip` - (Optional) Public IP address. The object structure is documented below.
+* `public_ip` - (Optional, List, ForceNew) Public IP address.The [public_ip](#dws_public_ip) object structure is
+    documented below. Changing this will create a new resource.
```

Comment:
```suggestion
* `public_ip` - (Optional, List, ForceNew) Public IP address. The [public_ip](#dws_public_ip) object structure is
  documented below. Changing this will create a new resource.
```